mod/ping.php

```diff
@@ -136,13 +136,9 @@ function ping_init(App $a)
 
 		$notifs = ping_get_notifications(local_user());
 
-		$condition = ["`unseen` AND `uid` = ? AND `contact-id` != ? AND (`vid` != ? OR `vid` IS NULL)",
-			local_user(), local_user(), Verb::getID(Activity::FOLLOW)];
-		$fields = ['id', 'parent', 'verb', 'author-name', 'unseen', 'author-link', 'author-avatar', 'contact-avatar',
```

Comment:
Aren't the fields supposed to be used in the `network_ping` hook call? What is the goal of this hook?

Reply:
There is no addon using this hook. So I suggest that when the hook is needed some day, then the needed fields are just added. But since this is only for counting stuff, I also considered removing that hook at all.